custodia/store/sqlite.py

```diff
@@ -110,8 +110,11 @@ def list(self, keyfilter=''):
                     continue
                 if not row[0].startswith(child_prefix):
                     continue
-                value.append(row[0][len(child_prefix):].lstrip('/'))
-
+                result = row[0][len(child_prefix):].lstrip('/')
+                if not row[1]:
```

Comment:
An application can store an empty string. Do we use NULL for containers? In that case, please do ```if row[1] is None```.